runhouse/resources/hardware/cluster.py

```diff
@@ -753,17 +754,28 @@ def install_packages(
                 package. (Default: ``None``)
             conda_env_name (str, optional): Name of conda env to install the package in, if relevant. If left empty,
                 defaults to base environment. (Default: ``None``)
+            force_sync_local (bool, optional): Whether to install the locally installed version of a package if found,
+                if the package already exists on the cluster. By default, the local version will be installed only if
+                the package does not already exist on the cluster. To force installing the local version in this case,
+                set to ``True``. (Default: ``False``)
```

Comment:
wonder if there is a way to make the documentation of the flag a bit more clear? 
How about:
Whether to install the locally installed version of a package ~~if found~~, if the package already exists on the cluster. By default, the local version will be installed only if the package does not already exist on the cluster. To force installing the local version in ~~this~~ in any (?) case, set to ``True``. (Default: ``False``)